lib/model/folder_summary.go

```diff
@@ -36,7 +46,7 @@ type folderSummaryService struct {
 	lastEventReqMut sync.Mutex
 }
 
-func newFolderSummaryService(cfg config.Wrapper, m model.Model) *folderSummaryService {
+func NewFolderSummaryService(cfg config.Wrapper, m Model, id protocol.DeviceID) FolderSummaryService {
```

Comment:
id not used?

Reply:
Good catch, it's worse: It should be assigned to the `folderSummary.id`, as that is used. That's something I'd expect a linter to catch (unused function param).